libs/application/templates/children-domicile-transfer/src/forms/ChildrenDomicileTransferForm.ts

```diff
@@ -56,6 +59,33 @@ export const ChildrenDomicileTransferForm: Form = buildForm({
         }),
       ],
     }),
+    buildSection({
+      id: 'otherParent',
+      title: 'Hitt foreldri',
+      children: [
+        buildMultiField({
+          id: 'informationAboutOtherParent',
+          title: 'Fylltu inn upplýsingar um hitt foreldrið',
+          description:  (application) => {
+            let parent = (application.externalData.parentNationalRegistry?.data as {
+              parent?: object
+            })?.parent as Parent
```

Comment:
I was wondering about this.

 I saw this done like this other places in the application app.

Is there a better way to do this? Any suggestions?